docs/learn/learn-async-backing.md

```diff
@@ -0,0 +1,93 @@
+---
+id: learn-async
+title: Asynchronous Backing
+sidebar_label: Asynchronous Backing
+description: A brief overview of asynchronous backing, and how it affects Polkadot's scalability.
+keywords: [parachains, slots, backing, parablock]
+slug: ../learn-async
+---
+
+Asynchronous backing is a mechanism that introduces a _pipeline_ for parachain **<->** relay chain
+communication. This pipeline will allow collators to include more data in parachain blocks while
+reducing latency (from 12 to 6 seconds for parablock validation).
+
+```mermaid
```

Comment:
The diagram is really cool but I do not undrstand it. It would be helpful to have a detailed description of what is going on, with definitions for each term within the figure. Think about that the reader must be able to fully understand the diagram just by looking at it and readign the figure description.

What are the key messages in the figure? Where is the improvemnt from the previous situation? How the current situation looks like?

Reply:
I will work on making it less ambiguous + add the "before", then explain why 👍

Reply:
Addressed